src/loans/kamino.py

```diff
@@ -280,14 +302,18 @@ def save_health_ratios(self) -> None:
             self._get_reserve_configs()
         with get_db_session() as session:
             for loan_entity in self.loan_entities.values():
-                loan_entity.update_positions_from_reserve_config(self.reserve_configs, self.token_prices)
+                loan_entity.update_positions_from_reserve_config(
+                    self.reserve_configs,
+                    self.token_prices,
+                    self.elevation_groups_to_liquidation_threshold
+                )
                 new_health_ratio = KaminoHealthRatio(
                     slot=int(self.last_slot),
                     user=loan_entity.obligation,
                     health_factor=loan_entity.health_ratio(),
                     std_health_factor=loan_entity.std_health_ratio(),
                     collateral=str(round(loan_entity.collateral_market_value(), 6)),
-                    risk_adjusted_collateral=str(round(loan_entity.collateral_market_value(), 6)),
+                    risk_adjusted_collateral=str(round(loan_entity.collateral_risk_adjusted_market_value(), 6)),
```

Comment:
did nt we want to keep it unadjusted?

Reply:
That was before we found out how it's really done 😅